### PR TITLE
Allow ISTEX Summary to receive a custom ISTEX query

### PR DESCRIPTION
**Pull request by Kmaschta**

### CODE REVIEW

**Comment by ThieryMichel** [comment]
e2e test fails (fix fixture)

File: src/app/js/formats/istexSummary/getIstexData.js [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
+
+const buildIstexQuery = (options = defaultQueryOptions) => {
+    const opts = { ...defaultQueryOptions, ...options };
+    const params = {
+        ...omit(opts, ['query']),
+        q: `(${opts.query})`,
+    };
+
+    return (
+        ISTEX_API_URL +
```
Comment (ThieryMichel) [inline src/app/js/formats/istexSummary/getIstexData.js]: please use template literal

File: src/app/js/formats/istexSummary/getIstexData.js [hunk 2]
```diff
@@ -103,100 +148,126 @@ export const parseOtherData = ({ response, error }) => {
...
 
-export const getIssueUrl = ({ issn, year, volume, searchedField }) => () => ({
-    url: `${ISTEX_API_URL}/document/?q=(${encodeURIComponent(
-        `${searchedField}:"${issn}" AND publicationDate:"${year}" AND ${getVolumeQuery(
-            volume,
-        )}`,
-    )})&facet=host.issue[*-*:1]&size=0&output=*`,
+export const getIssueUrl = ({ value, year, volume, searchedField }) => () => ({
+    url: buildIstexQuery({
+        query: `${getFilterQuery(
```
Comment (ThieryMichel) [inline src/app/js/formats/istexSummary/getIstexData.js:175]: How about having getFilterQuery construct the whole filter query. With additional publicationDate, volume and issue arg?

**Review by ThieryMichel — CHANGES_REQUESTED** [review]

File: src/app/js/formats/istexSummary/getIstexData.js (hunk 2 above)
Comment (Kmaschta) [reply src/app/js/formats/istexSummary/getIstexData.js:175]: Would be a nice addition if we change the code later.

**Review by ThieryMichel — APPROVED** [review]

**Pull request merged**